core/api/full_node/node_api.hpp

```diff
@@ -265,6 +266,17 @@ namespace fc::api {
     TokenAmount max_fee;
   };
 
+  struct MinerInfo {
+    Address owner;
+    Address worker;
+    std::vector<Address> control;
+    Buffer peer_id;
+    std::vector<Multiaddress> multiaddrs;
+    RegisteredPoStProof window_post_proof_type{RegisteredPoStProof::kUndefined};
```

Comment:
window_post_proof_type is used starting v3. For v0 & v2 we use `getRegisteredWindowPoStProof(seal_proof_type)`, is it ok for you?

Reply:
v0/v2 `MinerActorState::getInfo()` sets window_post_proof_type from seal_proof_type

Reply:
Yes, my question is: is it ok that getInfo() sets window_post_proof_type from seal_proof_type? Is it suitable for you in cases when api is called for v0 or v2?

Reply:
Yes, suitable.